mapper/views.py

```diff
@@ -0,0 +1,1278 @@
+        class_f_options = deepcopy(tree.d3_options)
+        class_t2_data = tree.get_tree_data(ProteinFamily.objects.get(name__startswith='Class T (Taste 2)'))
+        class_t2_options = deepcopy(tree.d3_options)
+        ### GETTING NODES
+        data_a = class_a_data.get_nodes_dict(None)
+        data_b1 = class_b1_data.get_nodes_dict(None)
+        data_b2 = class_b2_data.get_nodes_dict(None)
+        data_c = class_c_data.get_nodes_dict(None)
+        data_f = class_f_data.get_nodes_dict(None)
+        data_t2 = class_t2_data.get_nodes_dict(None)
+        #Collating everything into a single tree
+        general_options = {'depth': 4,
+                           'branch_length': {1: 'Class A (Rhodopsin)',
+                                             2: 'Alicarboxylic acid',
+                                             3: 'Gonadotrophin-releasing hormone',
+                                             4: ''},
+                           'branch_trunc': 0,
+                           'leaf_offset': 30,
+                           'anchor': "tree_plot",
+                           'label_free': [],
+                           'fontSize': {
+                                'class': "15px",
+                                'ligandtype': "14px",
+                                'receptorfamily': "13px",
+                                'receptor': "12px"
+                            }}
+        master_dict = OrderedDict([('name', ''),
+                                   ('value', 3000),
+                                   ('color', ''),
+                                   ('children',[])])
+        class_a_dict = OrderedDict([('name', 'Class A (Rhodopsin)'),
+                                   ('value', 0),
+                                   ('color', 'Red'),
+                                   ('children',data_a['children'])])
+        class_b1_dict = OrderedDict([('name', 'Class B1 (Secretin)'),
+                                   ('value', 0),
+                                   ('color', 'Green'),
+                                   ('children',data_b1['children'])])
+        class_b2_dict = OrderedDict([('name', 'Class B2 (Adhesion)'),
+                                  ('value', 0),
+                                  ('color', 'Blue'),
+                                  ('children',data_b2['children'])])
+        class_c_dict = OrderedDict([('name', 'Class C (Glutamate)'),
+                                  ('value', 0),
+                                  ('color', 'Purple'),
+                                  ('children',data_c['children'])])
+        class_f_dict = OrderedDict([('name', 'Class F (Frizzled)'),
+                                  ('value', 0),
+                                  ('color', 'Grey'),
+                                  ('children',data_f['children'])])
+        class_t2_dict = OrderedDict([('name', 'Class T (Taste 2)'),
```

Comment:
please change to new class name: Class T2 (Taste 2)

Reply:
Fixed this!